### PR TITLE
Move from npm to yarn

### PR DESCRIPTION
**Pull request by PowerKiKi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```
Comment (odahcam) [inline .travis.yml:15]: I'm not sure we **need** to install yarn for Travis, I think they deliver it with [their linux images](https://travis-ci.org/github/zxing-js/library/jobs/694760818#L204).

File: README.md [hunk 2]
```diff
@@ -34,12 +34,11 @@ Moreover, the component can shows name initials or simple value as avatar.
 
 ## Installation
 
-Install avatar component using [NPM](https://www.npmjs.com/)::
+Install avatar component using [Yarn](https://yarnpkg.com/)::
 
 ```bash
-$ npm install ngx-avatar --save
+$ yarn add ngx-avatar
```
Comment (odahcam) [inline README.md:40]: Maybe we should keep both examples, for convenience.

**Comment by odahcam** [comment]
Thanks for the PR @PowerKiKi .

File: .travis.yml (hunk 1 above)
Comment (PowerKiKi) [reply .travis.yml:15]: I thought so too, but actually build fails with a too old yarn: https://travis-ci.org/github/HaithemMosbahi/ngx-avatar/builds/694470294#L499

File: README.md (hunk 2 above)
Comment (PowerKiKi) [reply README.md:40]: done

File: .travis.yml (hunk 1 above)
Comment (odahcam) [reply .travis.yml:15]: That's odd, haha. I don't know how my build pipe works then, we can let it there for now anyways and improve if possible with time. Maybe Travis auto-includes yarn once it detects `yarn.lock`. Let's see what happens.

**Comment by odahcam** [comment]
Ready to merge once `package-lock.json` conflicts are solved, which means tell git to delete the file.

**Comment by PowerKiKi** [comment]
done via rebasing

**Comment by odahcam** [comment]
Thanks again @PowerKiKi

**Comment by PowerKiKi** [comment]
So shall we merge this ?

**Comment by odahcam** [comment]
I was waiting for more changes to pack a new version within this change and release it, but I got caught in the middle with personal matters that stole my spare time to do that. I think we should merge it now.

**Pull request merged**